.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow publish the GemWallet API to the npm repository
+name: Release
+
+on:
+  # Runnable manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version number"
+        type: "string"
+        required: true
+
+jobs:
+  call-workflow-integration:
+    name: Integration
+    uses: ./.github/workflows/integration.yml
+
+  release:
+    name: 🚀 Release
+    needs: call-workflow-integration
+    runs-on: ubuntu-latest
+    steps:
+      # Download api build artifact from integration workflow
+      - name: Download api-build artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: api-build
+          path: packages/api/dist
+
+      # Change version in package.json
+      - name: Change api version in package.json
+        run: |
+          echo "Setting api version to: ${{ github.event.inputs.version }}"
+          npm version ${{ github.event.inputs.version }} --no-git-tag-version
+        working-directory: packages/api/dist
+
+      # Publish to npm registry
+      - name: Publish to npm
+        run: npm run release:api:npm
```

Comment:
@FlorianBouron We should test it with another npm repo before using it with gemwallet-extension